pkg/expectations/expectations.go

```diff
@@ -125,18 +128,18 @@ type ControllerExpectations struct {
 
 // GetExpectations returns the ControlleeExpectations of the given controller.
 func (r *ControllerExpectations) GetExpectations(controllerKey string) (*ControlleeExpectations, bool, error) {
+	var err error
```

Comment:
Because the declaration in the if statement below is "in an if statement" so what happens is without an "else" the compiler sees it as possibly never set/initialized.

It's sort of one problem leading to another; typically returns in if/else blocks is wasteful, as opposed to just falling through to a default return; so fixing that means now there's not a guarantee of err being set (which in this logic the default return is when an error exists so it needs to be set) and as a result it needs to be declared.

You can check some of this out by just running `golint xxxx.go`